typeid/typeid-sql/supabase/tests/05_operator_test.sql

```diff
@@ -0,0 +1,27 @@
+-- Start transaction and plan the tests.
+BEGIN;
+SELECT plan(30);
```

Comment:
I'm guessing that this int value is a seed so it enables the tests to run independently of one another even though they're using the same table name, but if not and there's some additional semantic meaning to it, let me know and I can change this to whichever plan value that you'd like

Reply:
The number in `plan` is the number of tests that this test file is going to run; in this case `2`. I've gone ahead and updated it to `plan(2)`